### PR TITLE
Review bap 6592

### PR DESCRIPTION
**Pull request by xabbuh**
This finishes laboro/documentation#1.

### CODE REVIEW

File: cookbook/how_to_replace_inline-javascript_with_component.rst [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+.. seealso::
+
+    You can find more information about the Page Component in the
+    :doc:`"Frontend Architecture chapter" </book/frontend_architecture>` and in the
+    `Page Component documentation`_.
+
+Creating a Page Component Module
+--------------------------------
+
+A Page Component is a JavaScript object based on the ``BaseComponent``. The inline-JavaScript from
```
Comment (xabbuh) [inline cookbook/how_to_replace_inline-javascript_with_component.rst]: What do you think of this sentence instead of talking about a JavaScript class?

**Comment by xabbuh** [comment]
@Grygir I think I covered all of your comments.

File: cookbook/how_to_replace_inline-javascript_with_component.rst (hunk 1 above)
Comment (Grygir) [inline cookbook/how_to_replace_inline-javascript_with_component.rst]: Think, it is fine. Thanks.
It's hard to make better a sentence without ``class`` word, where ``class`` is the best fit for it.

**Comment by Grygir** [comment]
@xabbuh, the article looks great, thanks for you.

**Pull request merged**